_posts/2023-05-20-quarkus-3-upgrade.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: 'Migration to Quarkus 3.0 is a breeze'
+date: 2023-05-20
+tags: migration
+synopsis: 'Quarkus 3.0, our new major release, is out. Now is the time to migrate!'
+author: mkouba
+---
+
+Quarkus 3 is out and it's about the time to update your application!
```

Comment:
Suggested an improvement but I'm not sure we want to push this message. The update to Quarkus 3 might not be easy for people using Hibernate ORM so I wouldn't want people to feel pressured to do it.
```suggestion
Quarkus 3 is out and it's about time to update your application!
```

Reply:
Have you seen my comment about not pressuring people?